benchmarks/spack/kathleen/compute-node/spack.yaml

```diff
@@ -0,0 +1,320 @@
+    autoconf:
+      externals:
+      - spec: autoconf@2.69
+        prefix: /usr
+    automake:
+      externals:
+      - spec: automake@1.13.4
+        prefix: /usr
+#  binutils:
+#    externals:
+#    - spec: binutils@2.27.43
+#      prefix: /usr
+    bzip2:
+      externals:
+      - spec: bzip2@1.0.6
+        prefix: /usr
+    cmake:
+      externals:
+      - spec: cmake@2.8.12.2
+        prefix: /usr
+      - spec: cmake@3.2.1
+        prefix: /shared/ucl/apps/cmake/3.2.1/gnu-4.9.2
+      - spec: cmake@3.7.2
+        prefix: /shared/ucl/apps/cmake/3.7.2/gnu-4.9.2
+      - spec: cmake@3.13.3
+        prefix: /shared/ucl/apps/cmake/3.13.3/gnu-4.9.2
+      - spec: cmake@3.19.1
+        prefix: /shared/ucl/apps/cmake/3.19.1/gnu-4.9.2
+      - spec: cmake@3.21.1
+        prefix: /shared/ucl/apps/cmake/3.21.1/gnu-4.9.2
+#  coreutils:
+#    externals:
+#    - spec: coreutils@8.22
+#      prefix: /usr
+    curl:
+      externals:
+      - spec: curl@7.29.0+ldap
+        prefix: /usr
+    diff:
+      externals:
+      - spec: diff@3.3
+        prefix: /usr
+    find:
+      externals:
+      - spec: find@4.5.11
+        prefix: /usr
+    flex:
+      externals:
+      - spec: flex@2.5.39~lex
+        prefix: /shared/ucl/apps/flex/2.5.39/gnu-4.9.2
+    gawk:
+      externals:
+      - spec: gawk@4.0.2
+        prefix: /usr
+    git:
+      externals:
+      - spec: git@2.32.0+tcltk
+        prefix: /shared/ucl/apps/git/2.32.0/gnu-4.9.2
+      - spec: git@1.8.3.1~tcltk
+        prefix: /usr
+    gmake:
+      externals:
+      - spec: gmake@3.82
+        prefix: /usr
+    groff:
+      externals:
+      - spec: groff@1.22.2
+        prefix: /usr
+    hwloc:
+      externals:
+      - spec: hwloc@1.11.12
+        prefix: /shared/ucl/apps/hwloc/1.11.12
+    intel-mkl:
+      externals:
+      - spec: intel-mkl@2018.3.222
+        prefix: /shared/ucl/apps/intel/2018.Update3/compilers_and_libraries_2018.3.222/linux/mkl
+      - spec: intel-mkl@2020.0.166
+        prefix: /shared/ucl/apps/intel/2020/compilers_and_libraries_2020.0.166/linux/mkl
+    intel-mpi:
+      externals:
+      - spec: intel-mpi@2018.3.222
+        prefix: /shared/ucl/apps/intel/2018.Update3
+      - spec: intel-mpi@2020.0.166
+        prefix: /shared/ucl/apps/intel/2020
+    intel-oneapi-mpi:
+      externals:
+      - spec: intel-oneapi-mpi@2021.6.0
+        prefix: /shared/ucl/apps/intel/2022.2/
+    libtool:
+      externals:
+      - spec: libtool@2.4.2
+        prefix: /usr
+    libxml2:
+      externals:
+      - spec: libxml2@2.9.4
+        prefix: /shared/ucl/apps/libxml2/2.9.4/gnu-4.9.2
+    libxsmm:
+      externals:
+      - spec: libxsmm@1.16.1
+        prefix: /shared/ucl/apps/cp2k/8.2/cp2k-8.2/tools/toolchain/install/libxsmm-1.16.1
+    m4:
+      externals:
+      - spec: m4@1.4.16
+        prefix: /usr
+    ninja:
+      externals:
+      - spec: ninja@1.10.2.git.kitware.jobserver-1
+        prefix: /shared/ucl/apps/python/bundles/python39-6.0.0/venv
```

Comment:
Your python was complaining about this yesterday when you changed the version to 3.11, wasn't it? Does this package implicitly depend on python 3.9, as the name might suggest.

Reply:
No, it wasn't. It might if one uses the package, but I haven't...